dockerfiles/compile-extension/Dockerfile_alpine

```diff
@@ -0,0 +1,27 @@
+FROM alpine:3.5
+
+RUN mkdir -p /app
+WORKDIR /app
+
+ADD ./docker-php-source /usr/bin/docker-php-source
+ADD ./install-php /usr/bin/install-php
+
+ARG php_version
+ARG php_sha
+ARG php_api
+
+ENV SRC_DIR=/usr/src
+ENV PHP_VERSION=${php_version}
+ENV PHP_API=${php_api}
+ENV PHP_URL=https://secure.php.net/get/php-${php_version}.tar.xz/from/this/mirror
```

Comment:
I believe the preferred URL scheme changed to:
```suggestion
ENV PHP_URL=https://www.php.net/distributions/php-${php_api}.tar.xz
```

Reply:
it is `${php_version}` so I am not accepting this specific suggestion and push directly the change, instead